socorro/cron/jobs/featured_versions_automatic.py

```diff
@@ -0,0 +1,138 @@
+import re
+
+import requests
+
+from configman import Namespace
+from crontabber.base import BaseCronApp
+from crontabber.mixins import (
+    with_postgres_transactions,
+    with_single_postgres_transaction,
+)
+
+
+def alias_list_to_dict(input_string):
+    aliases = {}
+    for item in input_string.split(','):
+        if not item.strip():
+            continue
+        key, value = item.split(':')
+        aliases[key.strip()] = value.strip()
+    return aliases
+
+
+class DownloadError(Exception):
```

Comment:
No, I quite like the idea that they are different. They just happen to have the same name.
Also, the correlations.py code is also dying a slow death. 

Letting this module have its own class make it easy to reason about where the error is coming from.

If this exception does get instansiated, the crontabber app will simply fail, and log the error.